module/zcommon/zfs_prop.c

```diff
@@ -125,6 +127,99 @@ zfs_prop_init(void)
 		{ "gzip-9",	ZIO_COMPRESS_GZIP_9 },
 		{ "zle",	ZIO_COMPRESS_ZLE },
 		{ "lz4",	ZIO_COMPRESS_LZ4 },
+		{ "zstd",	ZIO_COMPRESS_ZSTD },
+		{ "zstd-fast",	ZIO_COMPRESS_ZSTD |
+		    (ZIO_ZSTDLVL_FAST_1 << SPA_COMPRESSBITS) },
```

Comment:
This should use the ZIO_ZSTD_FAST_LEVEL_DEFAULT macro, rather than being statically defined as level 1

Reply:
This is a good point, indeed it should...
@c0d3z3r0 Simple fix I guess? This was added after my request to set a default for zstd-fast...

Reply:
done, please check again and mark as resolved

Reply:
@c0d3z3r0 It currently throws a build error...
`../../module/zcommon/zfs_prop.c: In function ‘zfs_prop_init’:
../../module/zcommon/zfs_prop.c:131:8: error: ‘ZIO_ZSTDLVL_FAST_LEVEL_DEFAULT’ undeclared (first use in this function)
       (ZIO_ZSTDLVL_FAST_LEVEL_DEFAULT << SPA_COMPRESSBITS) },
        ^
../../module/zcommon/zfs_prop.c:131:8: note: each undeclared identifier is reported only once for each function it appears in
Makefile:1125: recipe for target 'zfs_prop.lo' failed
make[3]: *** [zfs_prop.lo] Error 1`

Reply:
typo alarm ;) fixed